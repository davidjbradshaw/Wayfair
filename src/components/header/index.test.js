import { render } from '@testing-library/react'

import Header from './index'

test('renders header', () => {
  const { container } = render(<Header />)
  expect(container).toMatchSnapshot()
})
