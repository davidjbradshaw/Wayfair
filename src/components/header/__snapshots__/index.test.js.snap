// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header 1`] = `
<div>
  <header
    style="color: rgb(127, 24, 127);"
  >
    <img
      alt="Wayfair - Partner Home"
      src="logo.svg"
      style="height: 28px; pointer-events: none; margin: 15px 20px 10px;"
    />
    <button
      aria-controls="simple-menu"
      aria-haspopup="true"
      class="MuiButtonBase-root MuiButton-root MuiButton-text"
      data-testid="account-menu-button"
      style="position: absolute; top: 14px; right: 0px; color: rgb(127, 24, 127);"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        <span
          class="material-icons md-24"
        >
          account_circle
        </span>
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </header>
</div>
`;
